frontend/src/AlertConfiguration/AlertConfigurationTests/AlertConfigurationSetupTest.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```

Comment:
ooc, are you going to be adding tests for the frontend code you wrote?

Reply:
You can ignore the test file.